eng/service.proj

```diff
@@ -13,7 +13,7 @@
     <IncludeSamplesApplications Condition="'$(ServiceDirectory)' != '*' or '$(IncludeSamples)' == 'false'">false</IncludeSamplesApplications>
   </PropertyGroup>
 
-  <ItemGroup>
+  <ItemGroup Condition="'$(TestProjectFileDependsOnCore)' == '' ">
```

Comment:
We should make this a more generic property like `$(ProjectListOverrideFile)`

Reply:
Done